config/db/seed.js

```diff
@@ -315,22 +443,61 @@ async function dropDb() {
 
 async function createParticipations() {
   const users = await User.find();
-  await Participation.create(
-    users.map(user => {
-      challenge: currentChallenge, user;
-    })
-  );
+  users.map(user => {
+    Participation.create({
+      challenge: currentChallenge,
+      user
+    });
+  });
+}
+
+async function createUsers() {
+  const families = await Family.find();
+  const users = require("./json/users.json");
+  users.map(user => {
```

Comment:
This is where you admin issue is failing. You don't ever assign it here.

Reply:
Oops

Reply:
Properly create users. Resolving.